mne/utils.py

```diff
@@ -2220,7 +2228,7 @@ def grand_average(all_inst, interpolate_bads=True, drop_bads=True):
                         else inst for inst in all_inst]
         equalize_channels(all_inst)  # apply equalize_channels
         from .evoked import combine_evoked as combine
-    elif isinstance(all_inst[0], AverageTFR):
+    else:  # isinstance(all_inst[0], AverageTFR):
```

Comment:
this will have to be `AverageTFR` because of previous tests (testing that all are either Evoked or AverageTFR and not entering if above - so not Evoked).